{
  "name": "context-error",
  "version": "0.0.0",
  "description": "Specifies where in the stack trace things went wrong and mentions in in the message",
  "main": "index.js",
  "scripts": {
    "test": "mocha"
  },
  "author": "Matthew Chase Whittemore <mcwhittemore@gmail.com>",
  "license": "BSD-2-Clause",
  "devDependencies": {
    "mocha": "1.14.0",
    "node-hooks": "0.0.15",
    "beautify.hks": "0.0.3",
    "should": "~2.0.2"
  },
  "directories": {
    "test": "test"
  },
  "dependencies": {
    "beautify.hks": "~0.0.3",
    "node-hooks": "~0.0.15",
    "should": "~2.0.2"
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/mcwhittemore/context-error.git"
  },
  "keywords": [
    "error",
    "context",
    "throw"
  ],
  "bugs": {
    "url": "https://github.com/mcwhittemore/context-error/issues"
  }
}
